CoffeeMachine_Task1/src/bg/coffee_machine/BasicCoffeeMachine.java

```diff
@@ -27,13 +27,15 @@ public void refill() {
         this.container.setCoffee(600);
         this.container.setWater(600);
     }
-    public boolean enoughEspresso(Beverage beverage){
-        return container.getCurrentCoffee() >= beverage.getCoffee() && container.getCurrentWater() >= beverage.getWater();
-    }
 
-    private void decreaseSuppliesForEspresso(Beverage beverage) {
+    private boolean enoughSupplies(Beverage beverage){
```

Comment:
There are duplicated functions (enoughSupplies & decreaseSupplies ) in Basic and Premium C.M. How to avoid it?